### PR TITLE
Xiao Improve new-user Team and Project setup efficiency

### PR DESCRIPTION
**Pull request by AuroraLovesWhales**
# Description
1. (PRIORITY MEDIUM) Jae: Improve new-user Team and Project setup efficiency
    1. Admin/Owner Login → Other Links → User Management → Create New User
    2. Right now, when I create a new user I have more clicks than I need for “Assign Project” and “Assign Team”
    3. What I’d like instead of what is shown below is to change the “Project” and “Team” tabs into “Assign Project’ and “Assign Team” buttons with the functions those buttons currently perform… eliminating the need for clicking the tabs and moving the information in those tabs so it is immediately visible with scrolling. 
unknown

## Main changes explained:
Removed nav component in UserProfileAdd.jsx and refactored the code to make "Assign Team" appear directly visible under "Assign Project".

## How to test:
    1. check into current branch
    2. do npm run start:local to run this PR locally
    3. log in as Admin/Owner, go to Other Links → User Management → Create New User
    5. verify "Assign Team" is immediately visible with scrolling

## Screenshots or videos of changes:
After:
Pasted Graphic

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/063c341b-8569-4ca7-8df8-31081bf01679

## Note:
Test to make sure the functionalities for "Assign Project" and "Assign Team" remain unchanged.

### CODE REVIEW

**Comment by one-community** [comment]
Looks good! Please fix all the alignment on the right: 
image

Delete buttons should be aligned right and the blue buttons should line up on the right too.

**Review by stevenpaalz — CHANGES_REQUESTED** [review]
Delete buttons are misaligned as stated in a prior comment. Also a scroll is needed to see these buttons, however I think that is inevitable with the size of the form.

Other than misaligned buttons, this is working as intended. Confirmed that correct functionality of both buttons still works.
Screenshot 2023-05-26 at 4 22 03 PM

**Review by NidaZaki — CHANGES_REQUESTED** [review]
Tested this PR and it works as intended. The functionality for both the buttons are correct. 

PR#850

As suggested in previous comments, the delete button for the team can be aligned to the right.

**Comment by AuroraLovesWhales** [comment]
I have adjusted the Delete button for Teams. I also added some padding to the "All", "Active", and "Complete" buttons to Tasks. However, I am unable to make the blue buttons ("Assign Project" and "Assign Team") line up on the right. Would appreciate some input on this. Thanks! Now the UI looks like this:
Pasted Graphic 1

**Review by stevenpaalz — DISMISSED** [review]
Delete button updates looks better. Thanks.

**Review by OAkinbode — CHANGES_REQUESTED** [review]
Hi, This idea and implementation look really good. I just have one question on this. What is that "Task" bar meant for when adding a new user?

New user process

I added a team and project I created that has a task assigned to the current account (sola test - admin). If tasks are attached primarily to individuals, will it be useful to create a task here and attach it to the new user? And if this will clog the page (I think it will) then we should drop the task section from the page altogether.
My team and project

I may be looking at this all wrong. So please correct me if I am wrong. 

Cheers,

**Review by EdwinLauM — DISMISSED** [review]
Works as intended! Suggestions were implemented as well! Good job! 

Screen Shot 2023-05-26 at 3 57 13 PM

**Comment by AuroraLovesWhales** [comment]
> Hi, This idea and implementation look really good. I just have one question on this. What is that "Task" bar meant for when adding a new user?
> 
> New user process
> I added a team and project I created that has a task assigned to the current account (sola test - admin). If tasks are attached primarily to individuals, will it be useful to create a task here and attach it to the new user? And if this will clog the page (I think it will) then we should drop the task section from the page altogether. My team and project
> 
> I may be looking at this all wrong. So please correct me if I am wrong.
> 
> Cheers,

Hi @OAkinbode! Thank you for your review. Actually I am not sure about the purpose of the Task section here. It is out of the scope of this PR. Maybe @one-community could confirm this?

**Review by yufeiStella — APPROVED** [review]
This is the safari version. The "Assign Team" appears as intended.
Screen Shot 2023-05-26 at 23 19 26

**Comment by one-community** [comment]
Good catch, yes, please remove the Tasks section. There is no point in showing that here.

**Comment by wang9hu** [comment]
Looking good! The Assign Team button and delete button work as intended! The new user will have the team assigned when created! Good Job!
Screenshot 2023-05-27 193456
Screenshot 2023-05-27 193537

**Review by 20chen-7 — CHANGES_REQUESTED** [review]
"style={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}" has appeared six times, maybe you should remove duplicated codes. The 'Task' is unnecessary, like previous suggestions, remove this part. Every function is intended.  Great jobs!
Screenshot 2023-05-30 at 12 24 59 PM

**Review by papiasharmin — DISMISSED** [review]
All functionalities  working as  intended. Great work.

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/86232710/746a7885-3314-4d2a-b29d-fffedca9366c

File: src/components/UserProfile/TeamsAndProjects/UserProjectsTable.jsx [hunk 1]
```diff
@@ -294,7 +297,10 @@ const UserProjectsTable = React.memo(props => {
               <span className="projects-span">Projects</span>
             </Col>
             {props.edit && props.role && (
-              <Col md="5" style={{ display: 'flex', alignItems: 'center', justifyContent: 'center'}}>
+              <Col
+                md="5"
+                style={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}
```
Comment (NidaZaki) [inline src/components/UserProfile/TeamsAndProjects/UserProjectsTable.jsx:302]: As pointed in one of the previous comments, maybe create a separate class with common styles so it is reusable.

**Comment by AnishPandita** [comment]
Hi @AuroraLovesWhales I tested your PR and the functionalities work as intended. The padding and indentation of the buttons also looks much cleaner and aligned. Great Job! Also i agree with the comments above to remove the duplicate code lines for 'style' tags.
Screenshot 2023-05-31 at 7 45 40 PM

**Review by lchua2314 — CHANGES_REQUESTED** [review]
Hi @AuroraLovesWhales I have three changes I would like you to do before I approve this PR:
1. Abstract the repeated code like what the others have requested.
2. Remove the Tasks section like what others have requested.
3. Please attempt to fix the package-lock.json if you're not making any changes with the dependencies. 
 
While I do request changes, the core functionality of the new feature works as intended. Good job!
Screenshot (2486)

File: src/components/UserProfile/TeamsAndProjects/UserProjectsTable.jsx [hunk 2]
```diff
@@ -332,7 +338,13 @@ const UserProjectsTable = React.memo(props => {
                       <td>{`${project.projectName}`}</td>
                       {props.edit && props.role && (
                         <td style={{ width: '103px' }}>
-                          <div style={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+                          <div
+                            style={{
+                              display: 'flex',
+                              alignItems: 'center',
+                              justifyContent: 'center',
```
Comment (eduhorta) [inline src/components/UserProfile/TeamsAndProjects/UserProjectsTable.jsx:345]: As suggested by other members, how about creating a separate class to facilitate reusability of this style?

**Review by eduhorta — APPROVED** [review]
It works as intended! Nice job! 👍

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/105796041/3396f87d-89f2-469c-9419-bf4280f1a7d5

**Review by Lucas-E — DISMISSED** [review]
I've reviewed your pull request
Captura de tela de 2023-06-03 16-28-24
 and I'm happy to confirm that it works as intended.

**Comment by AuroraLovesWhales** [comment]
> Looking good! The Assign Team button and delete button work as intended! The new user will have the team assigned when created! Good Job! Also since the toggle functionality is not used anymore, I think it is better to remove unused state and method like `activeTab`, `toggleTab()`. Screenshot 2023-05-27 193456 Screenshot 2023-05-27 193537

Hi @wang9hu! Thank you for your review. I have addressed your suggestions of removing `activeTab`, `toggleTab()`

**Comment by AuroraLovesWhales** [comment]
> "style={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}" has appeared six times, maybe you should remove duplicated codes. The 'Task' is unnecessary, like previous suggestions, remove this part. Every function is intended. Great jobs! Screenshot 2023-05-30 at 12 24 59 PM

Hi @20chen-7! Thank you for your review. I have addressed your suggestions of removing duplicated codes in styling and the 'Task' section
image

**Review by 20chen-7 — APPROVED** [review]
Now, it 
Screenshot 2023-06-06 at 6 55 46 PM
is really cool!

**Review by lchua2314 — APPROVED** [review]
Looks good now. Good job!

**Review by NidaZaki — APPROVED** [review]
Changes look good to me. Nice work!
image

**Review by wang9hu — APPROVED** [review]

**Comment by one-community** [comment]
Merged this because it had 4 approvals and I've got an urgent bug related to this function that I'm hoping this will fix.

**Pull request merged**